README.md

```diff
@@ -1,9 +1,7 @@
 .NET Compiler Platform ("Roslyn") Analyzers
```

Comment:
This change definitely looks worthwhile. Gen and I will be driving an overhaul of the roslyn-analyzer soon, one that will attempt to fix some factoring problems, make each set of projects consistent with others, etc. It'd be great to keep the rule docs updated against these improvements. Perhaps you could send your tool code my way? I will look at integrating it with some other tools Gen and I are producing to auto-generate analyzer infrastructure (based on a TSV description of checks).

This change also makes me think of our idea to create a common registry of checks to do things like reserve diagnostic id ranges, look for existing functionality, etc. Separate topic.



LGTM